src/olympia/lib/settings_base.py

```diff
@@ -939,8 +939,8 @@ def JINJA_CONFIG():
 
 # Default to short expiration; check "remember me" to override
 SESSION_ENGINE = 'django.contrib.sessions.backends.signed_cookies'
-SESSION_EXPIRE_AT_BROWSER_CLOSE = True
-SESSION_COOKIE_AGE = 1209600
+SESSION_EXPIRE_AT_BROWSER_CLOSE = False
+SESSION_COOKIE_AGE = 2592000 # github.com/mozilla/addons-server/issues/1789
```

Comment:
this will fail pep08, I'd put the comment before the actual object definition:

```python
# github.com/mozilla/addons-server/issues/1789
SESSION_COOKIE_AGE = ...
```